tests/unit/test_bugzilla.py

```diff
@@ -60,11 +61,43 @@ def test_lookup_action(actions_example):
     assert "test config" in action.description
 
 
-def test_lookup_action_missing(actions_example):
+def test_lookup_action_with_postfix(actions_example):
+    bug = bug_factory(id=1234, whiteboard="[example][DevTest-test]")
+    action = bug.lookup_action(actions_example)
+    assert action.whiteboard_tag == "devtest"
+    assert "test config" in action.description
+
+
+def test_lookup_action_with_prefix(actions_example):
+    bug = bug_factory(id=1234, whiteboard="[example][test-DevTest]")
+    action = bug.lookup_action(actions_example)
+    assert action.whiteboard_tag == "devtest"
+    assert "test config" in action.description
+
+
+def test_lookup_action_with_prefix_and_postfix(actions_example):
+    bug = bug_factory(id=1234, whiteboard="[example][foo-DevTest-bar]")
+    action = bug.lookup_action(actions_example)
+    assert action.whiteboard_tag == "devtest"
+    assert "test config" in action.description
```

Comment:
Since these 3 tests are constructed the same way, you can use `@pytest.parameterize` passing the whiteboard.